src/components/network-map-viewer/network/geo-data.ts

```diff
@@ -8,33 +8,49 @@
 import { computeDestinationPoint, getGreatCircleBearing, getRhumbLineBearing } from 'geolib';
 import cheapRuler from 'cheap-ruler';
 import { ArrowDirection } from './layers/arrow-layer';
+import { Line, LonLat } from '../utils/equipment-types';
+import { MapEquipments } from './map-equipments';
 
-const substationPositionByIdIndexer = (map, substation) => {
+export type Coordinate = {
+    lon: number;
+    lat: number;
+};
```

Comment:
It would require to make other changes in gridstudy-app, and in geo-data-server, conversions are also done to transform longiture into lon, and latitude into lat ...
So, maybe an harmonization could be done in another TS ...